apps/etterlatte-beregning/src/main/kotlin/beregning/grunnbeloep/GrunnbeloepRepository.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.etterlatte.beregning.grunnbeloep
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import no.nav.etterlatte.libs.common.objectMapper
+import java.io.FileNotFoundException
+import java.math.BigDecimal
+import java.time.YearMonth
+
+object GrunnbeloepRepository {
+    val historiskeGrunnbeloep: List<Grunnbeloep> =
+        objectMapper.readValue(readFile("/grunnbelop.json"), GrunnbeloepListe::class.java).grunnbeloep
+
+    private fun readFile(file: String) = GrunnbeloepRepository::class.java.getResource("/grunnbelop.json")?.readText()
+        ?: throw FileNotFoundException("Fant ikke filen $file")
+
+    fun hentGjeldendeGrunnbeloep(dato: YearMonth): Grunnbeloep {
```

Comment:
Dette brukes jo bare for opphør hvor det blir beregnet til 0, men er dette logikk som egentlig også bør/skal være regelspesifisert? Eller trenger vi ikke det tro 🤔

Reply:
Grunnbeløpet her brukes ikke til noe i manuelt opphør. Det vises i frontend da det er felles view for alle behandlingtyper her.